XamlControlsGallery/Helper/UIHelper.cs

```diff
@@ -1,12 +1,16 @@
 using System.Collections.Generic;
 using System.Linq;
+using Windows.Storage;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Media;
 
 namespace AppUIBasics.Helper
 {
     public static class UIHelper
     {
+        public static bool IsScreenshotMode { get; set; }
```

Comment:
Does it make sense to store this in the UIHelper?

Reply:
It's UI, and it's helping. :D

Reply:
You are technically correct, the best kind of correct!